src/common-components/RedirectLogistration.jsx

```diff
@@ -40,7 +41,7 @@ const RedirectLogistration = (props) => {
       setCookie('van-504-returning-user', true);
 
       if (registrationEmbedded) {
-        window.parent.postMessage({ action: REDIRECT, redirectUrl: `${window.location.origin}${AUTHN_PROGRESSIVE_PROFILING}` }, DISCOVER_URL);
+        window.parent.postMessage({ action: REDIRECT, redirectUrl: `${window.location.origin}${AUTHN_PROGRESSIVE_PROFILING}` }, host);
```

Comment:
Add the url in the edx-internal config
```suggestion
        window.parent.postMessage(
            {
                action: REDIRECT,
                redirectUrl: getConfig().POST_RESISTRATION_REDIRECT_URL
             },
            host
         );
```

Reply:
typo: POST_REGISTRATION_REDIRECT_URL